synapse/storage/schema/main/delta/73/02add_pusher_enabled.sql

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2022 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE pushers ADD COLUMN enabled BOOLEAN;
```

Comment:
Does this need to be nullable? Can you make this NOT NULL DEFAULT TRUE?

Reply:
I believe adding non-nullable fields locks the table, so we generally try not to do that.

Reply:
> I believe adding non-nullable fields locks the table, so we generally try not to do that.

This is no longer true from Postgres 11. Postgres 10 is EOL in 1 month :-)